src/components/Button/Button.tsx

```diff
@@ -33,7 +38,7 @@ export const Button: React.FC<ButtonProps> = ({
     <button
       type="button"
       className={cx(styles.button, {
-        [styles.orange]: orangeButton,
+        [styles[color!]]: true,
```

Comment:
`!` usually means you're doing something unsafe. If you want to make sure `color` really is defined, either make the parameter mandatory (remove the `?`), or use a default here:
```suggestion
        [styles[color ?? ButtonColor.orange]]: true,
```

Reply:
You are right. The `?` or `??` did not work satisfyingly. I wanted the color to be generic without defaulting to orange or green.

I am not used to React syntax yet, but this should do it: [a280673](https://github.com/BTE-Trusted-Entity/stakeboard/pull/74/commits/a2806730577b3162e6e53ac07a77c54c34d18703)